modules/tax/src/Plugin/Commerce/TaxType/CanadianSalesTax.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+namespace Drupal\commerce_tax\Plugin\Commerce\TaxType;
+
+use Drupal\commerce_order\Entity\OrderInterface;
+use Drupal\commerce_order\Entity\OrderItemInterface;
+use Drupal\commerce_tax\TaxableType;
+use Drupal\commerce_tax\TaxZone;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\profile\Entity\ProfileInterface;
+
+/**
+ * Provides the Canadian tax type.
+ *
+ * @CommerceTaxType(
+ *   id = "CanadianSalesTax",
+ *   label = "Canadian Sales Tax",
+ * )
+ */
+class CanadianSalesTax extends LocalTaxTypeBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
+    $form = parent::buildConfigurationForm($form, $form_state);
+    $form['rates'] = $this->buildRateSummary();
+    $form['rates']['#markup'] = $this->t('The following Canadian tax rates are provided:');
```

Comment:
Do we gain anything by having this whole override? The default text is "The following tax rates are provided:" which sounds descriptive enough.